api/tests/unit/app_analytics/test_migrate_to_pg.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+from app_analytics.migrate_to_pg import migrate_feature_evaluations
+from app_analytics.models import FeatureEvaluationBucket
+from django.conf import settings
+from django.utils import timezone
+from pytest_mock import MockerFixture
+
+
+@pytest.mark.skipif(
+    "analytics" not in settings.DATABASES,
+    reason="Skip test if analytics database is not configured",
+)
+@pytest.mark.django_db(databases=["analytics", "default"])
+def test_migrate_feature_evaluations(mocker: MockerFixture):
+    # Given
+    feature_name = "test_feature_one"
+    environment_id = "1"
```

Comment:
It feels odd to me that `environment_id` is set to a string here.

Reply:
Yeah, but unfortunately this is the data type our functions expects (historical reasons)